administration/src/bp-modules/UserMenu.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button, Menu, Popover } from '@blueprintjs/core'
+import React, { ReactElement, useContext } from 'react'
+import { NavLink, useNavigate } from 'react-router-dom'
+import styled from 'styled-components'
+
+import { WhoAmIContext } from '../WhoAmIProvider'
+import { roleToText } from './users/UsersTable'
+
+type UserMenuProps = {
+  isOpen: boolean
+  setIsOpen: (value: boolean) => void
+  onSignOut: () => void
+}
+
+const Backdrop = styled.div<{ isOpen: boolean }>`
+  display: ${props => (props.isOpen ? 'block' : 'none')};
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  z-index: 10;
+  left: 0;
+  top: 0;
+`
+
+const MenuContent = styled(Menu)`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+`
+
+const MenuItem = styled(Button)`
+  display: inline-block;
+  width: 100%;
+`
+
+const UserMenuButton = styled(Button)`
+  min-width: 220px;
+`
+
+const UserMenu = ({ isOpen, setIsOpen, onSignOut }: UserMenuProps): ReactElement => {
+  const { role, email } = useContext(WhoAmIContext).me!
+  const navigate = useNavigate()
+  const signOutAndRedirect = () => {
+    onSignOut()
+    navigate('/')
+  }
+  const userMenuContent = (
+    <MenuContent onClick={() => setIsOpen(false)}>
+      <span className='bp5-tag bp5-large'>{`Rolle: ${roleToText(role)}`}</span>
```

Comment:
Why not use `Tag` directly? In the current implementation this looks like a selected button. Don't like that.

Reply:
For me it looks like a badge? :/ tag is deprecated in the api docs

Reply:
what about this @sarahsporck 
<img width="273" alt="image" src="https://github.com/digitalfabrik/entitlementcard/assets/37902063/5dbab921-7233-455f-9cfa-428743a533cf">